common/dynamicconfig/constants.go

```diff
@@ -115,8 +115,6 @@ const (
 	DeadlockInterval = "system.deadlock.Interval"
 	// How many extra goroutines can be created per root.
 	DeadlockMaxWorkersPerRoot = "system.deadlock.MaxWorkersPerRoot"
-	// Enable / disable worker versioning.
-	EnableWorkerVersioning = "system.workerVersioning.enable"
```

Comment:
Is there anything deployed that will be broken when we remove this dynamic config? If not, then LGTM.

Reply:
This hasn't been in use yet so I'm not worried.